Classes/FairPlayDRM/FPSContentKeyManager.swift

```diff
@@ -0,0 +1,88 @@
+import AVFoundation
+
+fileprivate let skdUrlPattern = try! NSRegularExpression(pattern: "URI=\"skd://([\\w-]+)\"", options: [])
```

Comment:
Was this regex checked?...
<img width="1384" alt="screen shot 2018-05-29 at 4 19 16 pm" src="https://user-images.githubusercontent.com/5113371/40661359-3482dd6c-635c-11e8-88ed-7fd223d52c0a.png">
<img width="1384" alt="screen shot 2018-05-29 at 4 19 26 pm" src="https://user-images.githubusercontent.com/5113371/40661376-42b8a998-635c-11e8-8cfd-a8c78cfe37b5.png">

Reply:
Tested and works just fine.
<img width="1059" alt="image" src="https://user-images.githubusercontent.com/10047237/40685129-120e1e50-639c-11e8-9e48-76bbb28ab477.png">